src/utils/getTeams.ts

```diff
@@ -0,0 +1,14 @@
+import { PRODUCT_OWNERS_YML, GH_ORGS } from "@/config";
+
+export function getTeams(repo: string, productArea: string | undefined, org: string): string[] {
```

Comment:
I'm far from a typescript expert, but isn't the idiomatic spelling of an optional parameter `productArea?: string`? Or will that behave differently in some subtle way from what you have here?

Reply:
This is not very accurate as this field should never be undefined, and null is a type of string. Removing the `undefined` here.